src/gitManager/gitManager.ts

```diff
@@ -1,5 +1,6 @@
-import { App } from "obsidian";
+import { App, FileSystemAdapter } from "obsidian";
 import ObsidianGit from "../main";
+import { relative, join } from 'path';
```

Comment:
please import `path` with `as path` here to make the usage from the path library more clearly.

Reply:
Does it mean `import * as path from 'path';` ?